chain/epoch_manager/src/lib.rs

```diff
@@ -644,6 +653,27 @@ impl EpochManager {
     pub fn get_epoch_inflation(&mut self, epoch_id: &EpochId) -> Result<Balance, EpochError> {
         Ok(self.get_epoch_info(epoch_id)?.inflation)
     }
+
+    /// Compare two epoch ids based on their start height. This works because finality gadget
+    /// guarantees that we cannot two different epochs on two forks
+    pub fn compare_epoch_id(
```

Comment:
Why do we need this function? In my use case I only need `get_epoch_start_from_epoch_id`.

Reply:
Not really because for an epoch id you might not know its start height if it has not happened yet. Then you need to write a wrapper function which is basically what this function does.

Reply:
Makes sense.